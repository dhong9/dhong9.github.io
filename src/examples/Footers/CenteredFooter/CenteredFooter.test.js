import renderer from "react-test-renderer";
import CenteredFooter from "examples/Footers/CenteredFooter";

// Mocks
jest.mock("components/MKBox", () => {
    const { forwardRef } = jest.requireActual("react");
    return {
        __esModule: true,
        default: forwardRef(() => <div>Mock Box</div>)
    };
});
jest.mock("components/MKTypography", () => {
    const { forwardRef } = jest.requireActual("react");
    return {
        __esModule: true,
        default: forwardRef(() => <p>Mock Typography</p>)
    };
});
jest.mock("@mui/material/Link", () => {
    const { forwardRef } = jest.requireActual("react");
    return {
        __esModule: true,
        default: forwardRef(() => <div>MUI Link</div>)
    };
});
jest.mock("@mui/material/Grid", () => {
    const { forwardRef } = jest.requireActual("react");
    return {
        __esModule: true,
        default: forwardRef(() => <div>MUI Grid</div>)
    };
});
jest.mock("@mui/material/Stack", () => {
    const { forwardRef } = jest.requireActual("react");
    return {
        __esModule: true,
        default: forwardRef(() => <div>MUI Stack</div>)
    };
});

describe("CenteredFooter", () => {
    it("renders", () => {
        const component = renderer.create(
            <CenteredFooter>Test Centered Footer</CenteredFooter>
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});