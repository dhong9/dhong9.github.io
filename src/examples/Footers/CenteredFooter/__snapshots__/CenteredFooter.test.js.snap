// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenteredFooter renders 1`] = `
<div>
  Mock Box
</div>
`;
